aries-site/src/examples/components/avatar/AvatarSizePreviewExample.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { Avatar, Box, Text } from 'grommet';
+
+export const AvatarSizePreviewExample = () => (
+  <Box align="center" height="small" direction="row" gap="xlarge">
+    <AvatarContainer
+      alt="Eric Soderberg image"
+      size="small"
+      src="//s.gravatar.com/avatar/99020cae7ff399a4fbea19c0634f77c3?s=80"
+      title="Small avatar with profile picture"
+    ></AvatarContainer>
```

Comment:
```suggestion
    />
```
No children, can this just be self-closing?